addons/mail/static/src/components/rtc_call_participant_card/rtc_call_participant_card.xml

```diff
@@ -47,7 +47,7 @@
                             </t>
                         </span>
                         <div class="o_RtcCallParticipantCard_overlayTop">
-                            <t t-if="callParticipantCard.rtcSession.isMuted">
+                            <t t-if="callParticipantCard.rtcSession.isSelfMuted and !callParticipantCard.rtcSession.isDeaf">
```

Comment:
Out of scope of PR, more a remark on the whole templating of participant card overlay:

I'm not sure we want to combine some icons together. I see boolean expressions that are quite long, some of which show disjointed rendering for the icon overlay.
It's as if there was a lack of concept in modelling to reduce complexity of this template, e.g. having a field `RtcSession/status` of some sort.